db/controllers/controllers.js

```diff
@@ -37,9 +38,9 @@ exports.getArticleById = (req, res,next)=>{
 exports.getCommentsByArticleId = (req, res, next)=>{
     const {article_id}=req.params;
 
-    const promises=[fetchCommentsByArticleId(article_id),fetchArticleByIds(article_id)]
+    const promises=[fetchCommentsByArticleId(article_id),fetchArticleById(article_id)]
     Promise.all(promises)
-    .then(([comments,articleDetails])=>{
+    .then(([comments,articleData])=>{
         res.status(200).send({comments:comments});
     }).catch((err)=>{
         next(err)
```

Comment:
There's a few changes here relating to a different endpoint. Ensure that your commits are task specific, there shouldn't be any changes to do with any other tests/implentations.